protocols/s3/index.md

```diff
@@ -51,6 +51,12 @@ Transfer files to your [S3](http://aws.amazon.com/s3) account and browse the S3
 
 You must obtain the login credentials (Access Key ID and Secret Access Key) of your [Amazon Web Services Account](http://aws.amazon.com/account/) from the [*AWS Access Identifiers page*](https://console.aws.amazon.com/iam/home?#security_credential). Enter the *Access Key ID* and *Secret Access Key* in the login prompt.
 
+The default ports are:
+- HTTPS: 443
+- HTTP: 80
```

Comment:
Using HTTP connection only works when using optional _S3 (HTTP)_ connection profile. Using the Amazon S3 default connection profile the connection is always secured with TLS.